corehq/apps/hqadmin/templates/hqadmin/email/error_email.html

```diff
@@ -19,31 +19,24 @@
 
 {% for line in tb_list %}
 {% if forloop.counter0 == 0 %}
-<div>
-    <h3>{{ line }}</h3>
-</div>
+<h3>{{ line }}</h3>
 {% elif 'site-packages' in line %}
-<div>
-    <small>{{ line }}</small>
-</div>
+<pre style="color: #DDDDDD">{{ line }}</pre>
```

Comment:
Yup it renders fine. The screenshot is from google inbox, and I just checked that it works in normal gmail as well.